README.md

```diff
@@ -1,8 +1,36 @@
 ## Orange DAM PHP API client
 
-The [Orange DAM](https://www.orangelogic.com/products/digital-asset-management-system) APIs allow developers to safely and securely extend the capabilities of their Orange DAM platform. Through calls to its API endpoints, an Orange DAM site can be manipulated and integrated with other software.
+The [Orange DAM](https://www.orangelogic.com/products/digital-asset-management-system) APIs allow developers to extend the capabilities of their Orange digital asset management platform. Through calls to its API endpoints, an Orange DAM site can be manipulated and integrated with a CMS or other software/workflows.
 
-__NOTICE: This library is in pre-release and under active development. It should be considered a preview and not ready for production use.__
+This library powers the **[Orange DAM Drupal module](https://www.drupal.org/project/orange_dam)**. For integration with other CMS platforms, see [Future Development](#future-development).
+
+This library enables the following:
+
+- Authentication
+- Search content.
+- Media retrieval/import.
+- Access Keyword data/activity.
+- Retrieve media captions.
+- List deleted objects.
+
+### Future Development
+
+The Orange DAM API is extensive and this library just scratches the surface of available functionality. Some _ideas_ about where this library and/or the tools that consume it could go next include:
+
+- WordPress plugin.
+- Enable upload of content to Orange DAM.
+- Access file metadata.
+- Document support.
+- Create links between documents.
+- Integration with Figma asset browser.
+- Integration with Adobe Lightroom.
+- Integration with Adobe InDesign.
+- PDF conversion.
+- Extract waveforms from audio content.
+
+_NOTE:_ This library and the associated module(s) that depend on it provide the foundation for a custom integration with Orange DAM. Custom development work will be needed to create a data model in your CMS, build and run the migrations that map your data, as well as integrate with the events this module offers to allow for additional customization of the integration.
```

Comment:
We should remove mentions of migrations and modules here.

I think this whole `_NOTE_` should be removed or rewritten.

Reply:
Updating!